backend/fms_core/resources/container_move.py

```diff
@@ -29,17 +30,41 @@ class Meta:
     def before_import(self, dataset, using_transactions, dry_run, **kwargs):
         skip_rows(dataset, 6)  # Skip preamble and normalize dataset
 
+    def import_obj(self, obj, data, dry_run):
+        errors = {}
+
+        try:
+            # Test for container barcode to provide a better error message.
+            container = Container.objects.get(barcode=get_normalized_str(data, "Container Barcode to move"))
+        except Container.DoesNotExist as e:
+            errors["container"] = ValidationError([f"Container Barcode to move was not found."], code="invalid")
+            self.manuallyExclude.extend(["kind", "name", "barcode", "location", "coordinates"]) 
```

Comment:
Seems more or less convenient to me to exclude all the remaining fields if the container barcode is not found. Isnt there any other way to skip the cleaning of the fields? 

Also would rename the validation error 'Inexisting source container barcode' or something more concise :)